maven_iuvs/download.py

```diff
@@ -109,7 +109,7 @@ def call_rsync(remote_path,
                ssh_password,
                extra_flags=""):
     """
-    Updates the SPICE kernels by rsyncing the VM folders to the local machine.
+    Updates data (e.g., L1b data and SPICE kernels) by rsyncing the VM folders to the local machine.
```

Comment:
Thanks for noticing this!

Reply:
Yea, I'll try to fix comments if they are not completely correct or misleading, but it's better to check my English as always.

Reply:
This looks good to me